core/app/assets/js/ui/atoms/heading.js

```diff
@@ -10,9 +10,19 @@ export default function Heading( props ) {
 		classes.push( baseClassName + '-' + props.variant );
 	}
 
-	return React.createElement( tagName, {
-		className: classes.filter( ( classItem ) => classItem ).join( ' ' ),
-	}, props.children );
+	const classesString = Utils.arrayToClassName( classes );
+
+	if ( props.tag ) {
+		return React.createElement( tagName, {
+			className: classesString,
+		}, props.children );
+	}
+
+	return (
+		<h1 className={ classesString }>
```

Comment:
Why not use a default `props.tag=h1`?

Reply:
The storybook doesn't know how to get the component props when no element is returned. I understand it's unusual, but I think we need to consider it for the benefit of getting the component info automatically.

Reply:
`const Element = () => React.createElement...

return <Element...`

Reply:
Nice, fixed.